.github/workflows/release-dotnet.yml

```diff
@@ -0,0 +1,42 @@
+name: Release .NET
+
+on:
+  push:
+    branches: [ main ]
+
+env:
+  PROJECT_PATH: packages/dotnet/Cisco.Opentelemetry.Specifications
+  CSPROJ_PATH: packages/dotnet/Cisco.Opentelemetry.Specifications/Cisco.Opentelemetry.Specifications.csproj
+  PACKAGE_OUTPUT_DIRECTORY: ${{ github.workspace }}/output
+  NUGET_SOURCE_URL: https://api.nuget.org/v3/index.json
+
+jobs:
+  deploy:
```

Comment:
I just would rename this to release-dotnet to match the other cicd files (or change the other files..)

Reply:
true! will fix that to match :)